tests/test_api_urls.py

```diff
@@ -11,16 +11,22 @@
 import sys
 import unittest
 
+from mock import patch, MagicMock
+from requests import Response
+from werkzeug import wsgi
+
 # Add webcompat module to import path
 sys.path.append(os.path.realpath(os.pardir))
 import webcompat
 
+FIXTURES_PATH = os.getcwd() + '/tests/fixtures'
 
 # Any request that depends on parsing HTTP Headers (basically anything
 # on the index route, will need to include the following: environ_base=headers
 headers = {'HTTP_USER_AGENT': ('Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; '
                                'rv:31.0) Gecko/20100101 Firefox/31.0'),
-           'HTTP_ACCEPT': 'application/json'}
+           'HTTP_ACCEPT': 'application/json',
+           '_content': '{}'}
```

Comment:
I'm curious about this. What are you trying to achieve here?

`_content` is not a usual header among HTTP headers.